codebase/superdarn/src.bin/tk/tool/map_addimf.1.18/doc/map_addimf.doc.xml

```diff
@@ -23,7 +23,7 @@
 <option><on>-vb</on><od>verbose. Log information to the console.</od>
 </option>
 <option><on>-old</on><od>the input file is a <code>map</code> format file.</od></option>
-<option><on>-omni</on><od>use data from OMNI.</od>
+<option><on>-omni</on><od>use data from OMNI. (Currently not functional)</od>
```

Comment:
What is not functional?

Reply:
The -omni flag isn't functional.

Reply:
and to be very specific, the -omni flag is not functional because the ability to parse OMNI data was never implemented.  It's a good idea that has been low on the priority list.

Reply:
Maybe the `-omni` flag should be removed entirely since it has never actually been implemented.

Reply:
Sorry, what I meant to say is what is the function and why doesn't it work?

Reply:
There is a placeholder `load_omni` function in `map_addimf.c` which was added by @sshepherd which was intended to eventually allow reading of OMNI data, but was never completed and currently looks like this:

```
int load_omni()
{
  /* SGS: this does nothing right now */
  return (0);
}
```